(sync) Resolve conflicts with gatsby-i18n-source @ 79b09bc (#172)
gatsbyjs/gatsby-ja: docs/docs/mdx/programmatically-creating-pages.md

Gatsby is the best
```

> ## Generate スラッグ s

Reviewer: ```suggestion
## Generate slugs
```

Since MDX posts are being sourced outside of `src/pages`, each post

> needs to be given a スラッグ which tells Gatsby the URL to render to。

Reviewer: ```suggestion
needs to be given a slug which tells Gatsby the URL to render to.
```

```

If you skipped the last step and want to use frontmatter for your

> スラッグ s instead of the generated field, replace `fields` with `frontmatter`.

Reviewer: ```suggestion
slugs instead of the generated field, replace `fields` with `frontmatter`.
```
